CHANGELOG.md

```diff
@@ -114,6 +114,17 @@ In the future, the component will automatically use the use the `<a>` element if
 
 This change was introduced in [pull request #4646: Deprecate `element` parameter on button component](https://github.com/alphagov/govuk-frontend/pull/4646)
 
+#### Stop using `govuk-body-xs`, `govuk-!-font-size-14` and '14' on the typography scale
```

Comment:
we're trialling using 'type scale' instead of 'typography scale', so I'm changing for consistency across docs.

```suggestion
#### Stop using `govuk-body-xs`, `govuk-!-font-size-14` and '14' on the type scale
```